bundles/GlossaryBundle/config/pimcore/default.yaml

```
@@ -0,0 +1,15 @@
+# Pimcore
+pimcore:
+    glossary:
```

```suggestion
pimcore_glossary:
```
To be consistent, we probably should use a dedicated root key here, right? 
The config definition needs to be moved to the bundle as well. 
An we need to consider this in the upgrade notes.